packages/react-components/react-combobox/src/utils/Selection.types.ts

```diff
@@ -23,10 +23,8 @@ export type SelectionProps = {
   selectedOptions?: string[];
 };
 
-export type SelectionState = Required<Pick<SelectionProps, 'selectedOptions'>> & Pick<SelectionProps, 'multiselect'>;
-
 /* Values returned by the useSelection hook */
-export type SelectionValue = {
+export type SelectionState = {
```

Comment:
Update this because `SelectionValue` was an internally used type that now just duplicates parts of `SelectionProps` and `SelectionState`. It's cleaner to remove it and just use `SelectionState` in `ListboxState`/`ComboboxBaseState`.